test/expected/drop_chunks.out

```diff
@@ -1,5 +1,6 @@
 CREATE TABLE PUBLIC.drop_chunk_test1(time bigint, temp float8, device_id text);
 CREATE TABLE PUBLIC.drop_chunk_test2(time bigint, temp float8, device_id text);
+CREATE TABLE PUBLIC.drop_chunk_test3(time bigint, temp float8, device_id text);
```

Comment:
drop_chunk_test3 was never made into a hypertable with create_hypertable. Come to think of it, that should be an error in drop chunks. Can you change?

Reply:
Done